docs/monitoring-usage.rst

```diff
@@ -278,3 +278,86 @@ follows::
+available labels have values specified. The ``project-id`` label is a special case that should
+not be specified.
+
+>>> from gcloud import monitoring
+>>> # Create a Resource by its resource name, assign labels as dict
+>>> resource = client.resource('gce_instance', labels={
+...     'instance_id': '1234567890123456789',
+...     'zone': 'us-central1-f'
+... })
+>>> # Create a Metric using its metric descriptor. assigning label as dict
+>>> metric = client.metric(type='custom.googleapis.com/my_metric', labels={
+...      'status': 'successful'
+... })
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a ``Metric`` and ``Resource`` specified, the :class:`~gcloud.monitoring.client.Client`
+can be used to write :class:`~gcloud.monitoring.timeseries.Point` values.
+
+When writing points, the Python type of the value must match the *value_type* specified
+in the associated :class:`~gcloud.monitoring.metric.MetricDescriptor`. For example, a Python
+float will map to ``ValueType.DOUBLE``.
+
+Stackdriver Monitoring supports several **metric kinds**: *GAUGE*, *CUMULATIVE*, and *DELTA*.
+However, *DELTA* metrics can not be created as custom metrics.
+
+*GAUGE* metrics represent only a single point in time, so only the ``end_time`` should be
+specified::
+
+    >>> client.write_point(metric=metric, resource=resource, 3.14, end_time=end) # API call
+
+By default, end_time defaults to :meth:`~datetime.datetime.utcnow()`, so metrics written for the
+current time can be simplified to::
+
+   >>> client.write_point(metric, resource, 3.14) # API call
+
+For *CUMULATIVE* metrics, the same start time should be re-used repeatedly as more points are
+written to the time series. In this context, the start time is sometimes referred to as the *RESET*
+time. Once the cumulative metric is ready to be reset, the *RESET* time should be reset by
+specifying a new ``start_time``. Please refer to the `TimeSeries`_ documentation for more
+information on specifying time intervals.
+
+    >>> client.write_point(metric, resource, 3, start_time=RESET, end_time=TIME1) # API call
+    >>> client.write_point(metric, resource, 6, start_time=RESET, end_time=TIME2) # API call
+    >>> # Now reset the time series, for example on an application restart.
+    >>> RESET = datetime.utcnow()
+    >>> client.write_point(metric, resource, 3, start_time=RESET, end_time=TIME3) # API call
+
+To write multiple ``TimeSeries`` in a single batch, you can use
+:meth:`~gcloud.monitoring.client.write_time_series`::
+
+    >>> ts1 = client.time_series(metric1, resource1, 3.14, end_time=end_time)
+    >>> ts2 = client.time_series(metric2, resource2, 42, end_time=end_time2)
+    >>> client.write_time_series([ts1, ts2]) # API call
+
+While multiple time series can be written in a single batch, each ``TimeSeries`` object sent to
+the API must only include a single point.
+
+For more information on creating ``TimeSeries``, please refer to the API reference docs on
+`TimeSeries`_.
```

Comment:
Got rid of this one. But overall seems pretty harmless to link here and there /shruggie